README.md

```diff
@@ -1,6 +1,6 @@
-[简体中文](https://github.com/go-gitea/gitea/blob/master/README_ZH.md)
+[简体中文](README_ZH.md)
 
-<h1> <img src="https://raw.githubusercontent.com/go-gitea/gitea/master/public/img/gitea-192.png" alt="logo" width="30" height="30"> Gitea - Git with a cup of tea</h1>
+<h1> <img src="public/img/gitea-192.png" alt="logo" width="30" height="30"> Gitea - Git with a cup of tea</h1>
```

Comment:
This relative path looks suspicious.

Reply:
What do you mean?
i make it relative to
 * point respect branches
 * and it will work in mirrors too

Reply:
I must be mistaken about the intention of this PR, then. I thought that it was for Giteas's README.md? Currently Gitea's README is mainly found at:

`https://github.com/go-gitea/gitea`

So this relative image will be sought at:

`https://github.com/go-gitea/gitea/public/img/gitea-192.png` (not found)

Am I missing something here?

Reply:
@guillep2k the relative README.md is working fine
-> the png is set back to absolute